nextflow.config

```diff
@@ -88,6 +88,7 @@ params {
     // MALT
     run_malt                   = false
     malt_mode                  = 'BlastN'
+    malt_generatemegansummary = false
```

Comment:
Should the default not be `true`? We have agreed to standardize on some kind of tabular format, right?

Reply:
No, this is not a tabular file, but just an extra file that can be loaded into MEGAN (only afaik). The txt file is the thing we want, this is the default output of rma2infilo